esb/org.wso2.developerstudio.eclipse.gmf.esb.persistence/src/org/wso2/developerstudio/eclipse/gmf/esb/internal/persistence/BAMMediatorTransformer.java

```diff
@@ -22,68 +21,62 @@
 import org.apache.commons.lang.StringUtils;
 import org.apache.synapse.endpoints.Endpoint;
 import org.apache.synapse.mediators.base.SequenceMediator;
-import org.eclipse.core.runtime.Assert;
 import org.eclipse.emf.ecore.EObject;
+import org.wso2.carbon.mediator.bam.config.stream.StreamConfiguration;
 import org.wso2.developerstudio.eclipse.gmf.esb.BAMMediator;
 import org.wso2.developerstudio.eclipse.gmf.esb.EsbNode;
 import org.wso2.developerstudio.eclipse.gmf.esb.persistence.TransformationInfo;
 import org.wso2.developerstudio.eclipse.gmf.esb.persistence.TransformerException;
-import org.wso2.carbon.mediator.bam.config.stream.StreamConfiguration;
 
-public class BAMMediatorTransformer extends AbstractEsbNodeTransformer{
+public class BAMMediatorTransformer extends AbstractEsbNodeTransformer {
+
+    public void transform(TransformationInfo information, EsbNode subject) throws TransformerException {
+        information.getParentSequence().addChild(createBAMMediator(subject, information));
+
+        doTransform(information, ((BAMMediator) subject).getOutputConnector());
+    }
 
-	public void transform(TransformationInfo information, EsbNode subject) throws TransformerException {
-		information.getParentSequence().addChild(
-				createBAMMediator(subject, information));
+    public void createSynapseObject(TransformationInfo info, EObject subject, List<Endpoint> endPoints) {
+    }
 
-		doTransform(information,
-				((BAMMediator) subject).getOutputConnector());	
-	}
+    public void transformWithinSequence(TransformationInfo information, EsbNode subject, SequenceMediator sequence)
+            throws TransformerException {
+        sequence.addChild(createBAMMediator(subject, information));
+        doTransformWithinSequence(information, ((BAMMediator) subject).getOutputConnector().getOutgoingLink(), sequence);
 
-	public void createSynapseObject(TransformationInfo info, EObject subject,
-			List<Endpoint> endPoints) {	
-	}
+    }
 
-	public void transformWithinSequence(TransformationInfo information, EsbNode subject,
-			SequenceMediator sequence) throws TransformerException {
-		sequence.addChild(createBAMMediator(subject, information));
-		doTransformWithinSequence(information, ((BAMMediator) subject)
-				.getOutputConnector().getOutgoingLink(), sequence);
-		
-	}
-	
-	private org.wso2.carbon.mediator.bam.BamMediator createBAMMediator(
-			EsbNode subject, TransformationInfo information) throws TransformerException {
+    private org.wso2.carbon.mediator.bam.BamMediator createBAMMediator(EsbNode subject, TransformationInfo information)
```

Comment:
This variable name comes from EsbNodeTransformer interface which is implemented by this class. And I think that name came because of the EMF conventions were followed in the implementation.